projects/centeva-core/src/components/table/table.component.ts

```diff
@@ -33,22 +37,27 @@ export class TableComponent implements OnInit {
   get dataSource(): any {
     return this._dataSource;
   }
-  @Input() tableLoading = false;
+  @Input() tableLoading: boolean = false;
+  @Input() tableLoadingText: string = "Loading Results..";
+  @Input() isRowClickable: boolean = true;
   @Output() searchChanged = new EventEmitter<SearchCriteriaRequest>();
   @Output() rowSelected = new EventEmitter();
+  @Output() columnSelected = new EventEmitter<TableEmittedColumnClick>();
+  @Output() checkboxSelected = new EventEmitter<TableEmittedCheckboxClick>();
 
   public SortStates = SortStates;
   public ColumnSortState: IColumnSortState = {SortDirection : this.SortStates.NONE, SortProperty: ''}
   public ColumnDataTypes = ColumnDataTypes;
-  public columnNames = [];
+  public columnNames: any[] = [];
```

Comment:
I would try to put better types on things. `any` should be reserved as a last resort.